src/CacheManager.Microsoft.Extensions.Caching.Memory/MemoryCacheHandle`1.cs

```diff
@@ -87,6 +89,26 @@ public override bool Exists(string key, string region)
             return _cache.Contains(GetItemKey(key, region));
         }
 
+        /// <inheritdoc />
+        public override IEnumerable<string> Keys(string pattern, string region)
+        {
+            var keys = _cache.ListChildren(region ?? GetType().Name)
+                .Select(k => ParseKey(region));
+
+            if (pattern == "*")
+            {
+                return keys;
+            }
+            if (pattern.Contains("*") || pattern.Contains("?"))
```

Comment:
Just that I hadn't revisited the code, as I was focused on Dictionary and Redis.
I removed the implementation, as I don't have the resources to test it at the moment.